core/psh/sysexec.c

```diff
@@ -25,6 +27,63 @@ void psh_sysexecinfo(void)
 }
 
 
+static int psh_sysexec_wldargmatch(char *wld, int argc, char **argv, char wldend, int argmaxlen)
+{
+	int len = 0, ii;
+	char *currc = wld, *nextc, *cmdend;
+
+	cmdend = strchr(wld, wldend);
+	currc = wld;
+	for (ii = 0; ii < argc; ii++) {
+
+		if ((nextc = strchr(currc, ' ')) == NULL || nextc > cmdend)
+			nextc = cmdend;
+		if ((nextc == cmdend) ^ (ii == (argc - 1))) /* exits if: last argc XOR last wldcrd argument */
+			return -1;
+
+		len = nextc - currc;
+		if (len > argmaxlen || len == 0 || strlen(argv[ii]) != len || memcmp(argv[ii], currc, len) != 0)
+			return -1;
+		if (*nextc != wldend && nextc != NULL)
+			currc = nextc + 1;
+	}
+	
+	return 1;
+}
+
+
+static int psh_sysexec_checkcommand(int argc, char **argv)
+{
+	int maxcdmlen = 80, acc = -1;
```

Comment:
Shouldn't it be `maxcmdlen` instead of `maxcdmlen` ?

Reply:
Yes, it should. Letters swapped. Thanks